Cargo.toml

```diff
@@ -7,32 +9,39 @@ members = [
 	"clap_mangen",
 ]
 
+[workspace.package]
+categories = ["command-line-interface"]
+edition = "2018"
```

Comment:
FYI While there are cases where people might want different editions in a workspace, I think its a good idea to generally consolidate and share it.  Similar for `rust- version`.